docs/book/v2/table.md

```diff
@@ -1,4 +1,4 @@
-# Using Swoole Tables In Your Application
+# Using Swoole Tables in your Application
```

Comment:
The previous casing was correct. The changes proposed don't make sense — it should either use Title Casing, or it should use sentence casing; the changes are a mix. In Title Casing, only articles (a, an, the) are lowercase.
```suggestion
# Using Swoole Tables In Your Application
```

Reply:
First: Thanks for your comment and the hint which style you normally use. The mixing is wrong here; I have mistakenly inserted my own linguistic background here. 🤦‍♂ 

At the moment we have no style guide for the documentation but I have already prepared some points to create a new one.
The current status contains a simple capitalisation style like "sentence case" or "first letter". Both need no further explanation and the rules are low.
After the move to Laminas we can discuss this in a separate issue repot.

Reply:
I tend to agree that we should likely do Sentence case, as it will be the easiest for non-native English speakers to comply with.

Reply:
I will write down your suggestion. Thanks!